.config/starte2e.sh

```diff
@@ -10,6 +10,9 @@ export PATH="${PATH}:/usr/bin"
 touch tests/environment/zitadel/service-user.json
 chmod a+rw tests/environment/zitadel/service-user.json
 
+# Generate certificates
+tests/environment/certs/generate-certs.sh
+
 # Shut down any still running test-setup first
-docker compose --project-directory ./tests/environment down -v test-setup || true
+docker compose --project-directory ./tests/environment down -v || true
```

Comment:
This is deliberate to avoid having to restart the more expensive services. It saves ~20 seconds per test run on my end. The only thing that sometimes gets stuck between reboots IME is the docker network, we should delete that one explicitly.

Reply:
If I don't restart the ldap service it don't get the new certificates. I think now it should be fine

Reply:
ldap is also pretty slow to start; could we check the mtime on the certs or such to determine whether to restart?

Reply:
Now I have the script checking the mtime and only generating it if it's older than 25 days. See if it's better.

Reply:
Yep, ta!